src/main/java/roboy/linguistics/Question.java

```diff
@@ -0,0 +1,9 @@
+package roboy.linguistics;
+
+import java.util.List;
+
+public class Question {
```

Comment:
in class question you have a list of questions 
please give meaningful names to classes
add java doc comments